exporter/splunkhecexporter/heartbeat.go

```diff
@@ -0,0 +1,154 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkhecexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter"
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"sync"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
```

Comment:
opencensus is really old, there might be a more modern way to do this.

Reply:
Ideally we want to use open telemetry framework but our application is on open census for now since initially when we onboarded otel collector the otel metrics wasn't as mature. We do plan to migrate over once it is mature enough with 1.0 release.

Reply:
I don't see a good path to involve opencensus here.
How about we make the metrics PR a separate PR from the heartbeat impl?

Reply:
I'm not sure how this is usually done. Maybe worth asking to a wider audience.